src/js/actions/libraries.js

```diff
@@ -886,19 +879,10 @@ define(function (require, exports) {
      * @return {Promise}
      */
     var handleCompletePlacingGraphic = function () {
-        return this.dispatchAsync(events.libraries.PLACE_GRAPHIC_UPDATED, { isPlacing: false })
-            .bind(this)
-            .then(function () {
-                var policyStore = this.flux.store("policy");
-                
-                if (policyStore.areAllSuspended) {
-                    return this.transfer(policyActions.restoreAllPolicies);
-                }
-            });
+        return this.dispatchAsync(events.libraries.PLACE_GRAPHIC_UPDATED, { isPlacing: false });
```

Comment:
We should probably clear out these events as their sole purpose was to suspend/restore the policies. I will make the change once this is merged.